velox/exec/Task.cpp

```diff
@@ -2248,9 +2250,27 @@ std::string Task::toString() const {
   }
 
   if (numRemainingDrivers > 0) {
-    out << "drivers:\n";
+    bool addedCaption{false};
     for (auto& driver : drivers_) {
       if (driver) {
+        if (!addedCaption) {
+          out << "drivers:\n";
+          addedCaption = true;
+        }
+        out << driver->toString() << std::endl;
+      }
+    }
+  }
+
+  if (!driversClosedByTask_.empty()) {
+    bool addedCaption{false};
+    for (auto& driverShadow : driversClosedByTask_) {
+      auto driver = driverShadow.lock();
+      if (driver) {
+        if (!addedCaption) {
+          out << "zombie drivers:\n";
+          addedCaption = true;
+        }
         out << driver->toString() << std::endl;
```

Comment:
Do we want to log use_count here? Thanks!

Reply:
We can, it would just likely delay the whole process of landing this, because likely some tests would fail due to the changed string and it is not feasible to predict which ones.

Reply:
Added, we don't have a test that expects this part in a certain way.